tests/measurements/test_state.py

```diff
@@ -365,22 +364,6 @@ def func(x):
         ):
             d_func(pnp.array(0.1, requires_grad=True))
 
-    def test_no_state_capability(self, monkeypatch):
```

Comment:
We shouldn't be removing tests. Instead of removing this test. You can test it with `DefaultQubitLegacy`

Reply:
Same for any other unit tests have been removed for similar reasons.

Reply:
Fixd at the three commits starting from https://github.com/PennyLaneAI/pennylane/pull/6684/commits/83c0a35fe96ab0f297ee6d39c51d4660f6903152